modules/apps/headless/headless-workflow/headless-workflow-api/src/main/java/com/liferay/headless/workflow/resource/HeadlessWorkflowResource.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.workflow.resource;
+
+import com.liferay.headless.workflow.dto.WorkflowLog;
+import com.liferay.headless.workflow.dto.WorkflowLogCollection;
+import com.liferay.headless.workflow.dto.WorkflowTask;
+import com.liferay.headless.workflow.dto.WorkflowTaskCollection;
+import com.liferay.oauth2.provider.scope.RequiresScope;
+
+import javax.annotation.Generated;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+
+/**
+ * To access this resource, run:
+ *
+ *     curl -u your@email.com:yourpassword -D - http://localhost:8080/o/headless-workflow/1.0.0
+ *
+ * @author Javier Gamarra
+ * @generated
+ */
+@Generated("")
+@Path("/1.0.0")
+public interface HeadlessWorkflowResource {
+
+	@GET
+	@Path("/roles/{parent-id}/workflow-tasks")
+	@Produces({"*/*"})
+	@RequiresScope("headless-workflow-application.read")
+	public WorkflowTaskCollection<WorkflowTask> getRolesWorkflowTasks(
+			@QueryParam("page") Integer page,
+			@QueryParam("per_page") Integer perPage,
+			@PathParam("parent-id") String parentId)
+		throws Exception;
```

Comment:
Hey @petershin @brianchandotcom,

First of all, thanks for all the great work you are doing on the generator :)

Seeing this PR, I've noticed that these query params are being read from the openapi.yml and that's great, but in this case, the arguments `page` and `per_page` together are `Pagination` which we already had in past version of the generator.

Could it be generated in the following way instead:

```
@GET
@Path("/roles/{parent-id}/workflow-tasks")
@Produces({"*/*"})
@RequiresScope("headless-workflow-application.read")
public WorkflowTaskCollection<WorkflowTask> getRolesWorkflowTasks(
		@Context Pagination pagination,
		@PathParam("parent-id") String parentId)
	throws Exception;
```
Replacing both page and per_page for the Pagination.

Thanks a lot!

By the way, are you ok with this way of providing feedback about the generator? Do you prefer a PR or any other way?